docs/security/shiroauthentication.md

```diff
@@ -31,7 +31,11 @@ When you connect to Apache Zeppelin, you will be asked to enter your credentials
 ## Security Setup
 You can setup **Zeppelin notebook authentication** in some simple steps.
 
-### 1. Secure the HTTP channel
+### 1. Enable Shiro
+By default in `conf`, you will find `shiro.ini.template`, this file is used as an example and it is strongly recommended
+to create a `shiro.ini` file instead by doing the following command line `copy conf/shiro.ini.template conf/shiro.ini`.
+
```

Comment:
I think this part might need to be changed "instead by doing the following command line `copy conf/shiro.ini.template conf/shiro.ini`."  -> "by doing the following command line `cp conf/shiro.ini.template conf/shiro.ini`."